CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Welcome to Capture contributing guide 
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on [Capture](https://github.com/khattakdev/capture) :sparkles: .
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md). Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+
+## Getting started
+
+Before making code contributions, make sure to set up your development environment. Please follow the project's documentation for specific instructions.
+
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with the Capture, search if an issue already exists. If a related issue doesn't exist, you can open a new issue using a relevant issue form.
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/khattakdev/capture/issues) to find one that interests you. You can narrow down the search using `labels` as filters. If you find an issue to work on, you are welcome to open a PR with a fix.
+
+### Make Changes
+
+#### Make changes in a codespace
+
+Make your code changes on this branch. Ensure your changes adhere to our coding guidelines. 
```

Comment:
Add more info on how can one use codespace to make changes.

Reply:
I have made the required changes, please check them.